### PR TITLE
Us129279 aq setting page

### PR DESCRIPTION
**Pull request by JikaiLong**
Functional Testing

- [x] Browsers (Chrome, FF, Edge)
- [x] Responsive
- [x] Test cases
- [x] 'Settings' page display correct header and footer contents and has two tabs to navigate between quiz detail settings and quiz item settings, auto select is enabled where the corresponding tab is selected when user enters from either main dashboard or the quiz item dashboard 
Security, devops, perf: N/A
UX and docs
- [x]  Demo to Kevin

Screenshots:
image

@Brightspace/draco-vis

### CODE REVIEW

File: locales/en.js [hunk 1]
```diff
@@ -56,6 +56,11 @@ export default {
 	"quizItems:null" : "NULL",
 	"quizItem:na": 'N/A',
 
-	"settings:title": 'Settings'
-
+	"settings:title": 'Settings',
+	"settings:description": 'Set which metrics display in analysis and what the threshold to trigger as flagged item. If no threshold is desired, leave field as blank for default.',
```
Comment (Vitalii-Misechko) [inline locales/en.js:60]: Your screenshot looks good but visual-diff shows some odd font for this description

image

Comment (JikaiLong) [reply locales/en.js:60]: Fixed. I didn't apply the h3 font size to h2

File: pages/settings-view.js [hunk 2]
```diff
@@ -65,16 +84,51 @@ class SettingsView extends SkeletonMixin(Localizer(MobxLitElement)) {
...
-				<h1 class="d2l-heading-1">${this.localize('settings:title')}</h1>
+			<div class="d2l-insights-settings-page-main-container">
+				<div class="d2l-insights-settings-page-main-content">
+						<h1 class="d2l-heading-1">${this.localize('settings:title')}</h1>
+						<h2 class="d2l-heading-2">${this.localize('settings:description')}</h2>
+					<d2l-tabs>
+						<d2l-tab-panel text="${this.localize('settings:quizDetailsMetrics')}" ?selected = ${window.location.search.includes('from=main')}>
+							<div>Quiz Details Settings Placeholder</div>
+						</d2l-tab-panel>
+						<d2l-tab-panel text="${this.localize('settings:quizItemsMetrics')}" ?selected = ${window.location.search.includes('from=quiz')}>
```
Comment (NicholasHallman) [inline pages/settings-view.js:105]: What happens if neither tab is selected?

Comment (JikaiLong) [reply pages/settings-view.js:105]: Then by default, the quiz detail metrices is selected.

Comment (NicholasHallman) [reply pages/settings-view.js:105]: Is that something that tab-apenl manages if there are no selected? neat!

**Review by Vitalii-Misechko — APPROVED** [review]

**Review by NicholasHallman — APPROVED** [review]

**Pull request merged**